core/server/data/migrations/versions/4.33/02-add-default-free-product.js

```diff
@@ -0,0 +1,37 @@
+const {createTransactionalMigration} = require('../../utils');
+const ObjectID = require('bson-objectid');
+const {slugify} = require('@tryghost/string');
+const logging = require('@tryghost/logging');
+
+module.exports = createTransactionalMigration(
+    async function up(knex) {
+        const [result] = await knex
+            .count('id', {as: 'total'})
+            .from('products')
+            .where('type', 'free');
+
+        if (result.total !== 0) {
+            logging.warn(`Not adding default free tier, a free tier already exists`);
+            return;
+        }
+
+        const name = 'Free';
+
+        logging.info(`Adding tier "${name}"`);
+        await knex('products')
+            .insert({
+                id: ObjectID().toHexString(),
+                name: name,
+                description: 'Free Preview',
```

Comment:
The fixture is missing the `description` column data

Reply:
We decided to not have any default description for free tier, updated to remove from here and kept fixtures as-is. Good catch, thanks.